src/testUtils/TestRig.react.js

```diff
@@ -57,9 +59,14 @@ export default class TestRig {
     var element = this.domNode.find(selector)
     this.fillInElement(element, value)
   }
-  toggleCheckbox(selector) {
-    var element = this.domNode.find(selector) // Selector should be specific to the checkbox
-    $(element).prop('checked', !($(element)[0].checked))
+  toggleElement(selector) {
```

Comment:
@BJK I agree with the naming convention and separation of the methods but I don't understand why we will want to pass label text of the checkbox and not just use the selector, especially since the checkbox might be wrapped in a label like we are doing now in webapps.